components/01-atoms/buttons/twig/button.twig.test.js

```diff
@@ -12,7 +12,7 @@ describe('Twig Button', () => {
       join(__dirname, 'button.twig'),
       loadYaml(join(__dirname, 'button.yml')),
     );
-    expect(container).toMatchInlineSnapshot(`
+    expect(container).toMatchSnapshot(`
```

Comment:
You'll want to remove the inline snapshot here, so you're simply calling `expect(container).toMatchSnapshot()`.

Reply:
ah, ok